Gems/PhysX/Code/Editor/KinematicDescriptionDialog.cpp

```diff
@@ -0,0 +1,117 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include "KinematicDescriptionDialog.h"
+
+AZ_PUSH_DISABLE_DLL_EXPORT_MEMBER_WARNING
+#include <Editor/ui_KinematicDescriptionDialog.h>
+AZ_POP_DISABLE_DLL_EXPORT_MEMBER_WARNING
+
+static constexpr AZStd::string_view dynamicDescription = "With <b>Dynamic</b> rigid bodies, you can physics forces "
```

Comment:
` you can physics forces` -> ` you can use physics forces`

Reply:
+1